bitnami/pinniped/templates/supervisor/ingress.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.supervisor.ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ include "pinniped.supervisor.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.supervisor.ingress.annotations }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.supervisor.ingress.annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+spec:
+  {{- if and .Values.supervisor.ingress.ingressClassName (eq "true" (include "common.ingress.supportsIngressClassname" .)) }}
+  ingressClassName: {{ .Values.supervisor.ingress.ingressClassName | quote }}
+  {{- end }}
+  rules:
+    {{- if .Values.supervisor.ingress.hostname }}
+    - host: {{ .Values.supervisor.ingress.hostname | quote }}
+      http:
+        paths:
+          {{- if .Values.supervisor.ingress.extraPaths }}
+          {{- toYaml .Values.supervisor.ingress.extraPaths | nindent 10 }}
+          {{- end }}
+          - path: {{ .Values.supervisor.ingress.path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ .Values.supervisor.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "https" "context" $)  | nindent 14 }}
+    {{- end }}
+    {{- range .Values.supervisor.ingress.extraHosts }}
+    - host: {{ .name | quote }}
+      http:
+        paths:
+          - path: {{ default "/" .path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" $) }}
+            pathType: {{ default "ImplementationSpecific" .pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" $) "servicePort" "https" "context" $) | nindent 14 }}
+    {{- end }}
```

Comment:
`ingress.extraRules` is missing:

https://github.com/bitnami/charts/blob/master/template/CHART_NAME/templates/ingress.yaml#L49

Reply:
Added